extensions/azdata/src/common/httpClient.ts

```diff
@@ -17,13 +17,32 @@ export namespace HttpClient {
 	 * Downloads a file from the given URL, resolving to the full path of the downloaded file when complete
 	 * @param downloadUrl The URL to download the file from
 	 * @param targetFolder The folder to download the file to
-	 * @returns Full path to the downloaded file
+	 * @returns a promise to a full path to the downloaded file
 	 */
-	export function download(downloadUrl: string, targetFolder: string): Promise<string> {
+	export function downloadFile(downloadUrl: string, targetFolder: string): Promise<string> {
+		return download(downloadUrl, targetFolder);
+	}
+
+	/**
+	 * Downloads the text contents of the document at the given URL, resolving to a string containing the text when complete
+	 * @param url The URL of the document whose contents need to be fetched
+	 * @returns a promise to a string that has the contents of document at the provided url
+	 */
+	export async function getTextContent(url: string): Promise<string> {
+		Logger.log(loc.gettingTextContentsOfUrl(url));
+		return await download(url);
+	}
+
+	/**
+	 * @returns Full path to the downloaded file.
+	 * If the targetFolder is not defined then it returns the contents of the downloaded file.
```

Comment:
Please use the @param annotations for the parameters - see other functions for examples.